packages/drawer/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import { DefaultDrawerConfiguration } from './drawer-configuration';
+import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
+
+export function configure(frameworkConfig: FrameworkConfiguration, callback?: (config: DefaultDrawerConfiguration) => void) {
+  frameworkConfig.globalResources([
+    PLATFORM.moduleName('./resources/attach-focus-drawer-attribute'),
```

Comment:
Similar to our recent issue with ux-form, we should just use static resources for custom attribute. It's better for optimization, since we already got the decorators on those.

Reply:
Cool. I'll keep this in mind. I'm planning to do some refactoring here to allow for choosing to register this resource or not from plugin config.